packages/notifi-react-card/lib/components/subscription/subscription-card-views/InputFields.tsx

```diff
@@ -0,0 +1,186 @@
+    NotifiTwitterInput?: NotifiTwitterInputProps['classNames'];
+    NotifiSmsInput?: NotifiSmsInputProps['classNames'];
+    NotifiTelegramInput?: NotifiTelegramInputProps['classNames'];
+  }>;
+};
+export const InputFields: React.FC<InputFieldsProps> = ({
+  data,
+  classNames,
+  inputSeparators,
+  inputLabels,
+  allowedCountryCodes,
+  hideStartIcons,
+  inputDisabled,
+}) => {
+  return (
+    <>
+      {data.contactInfo.email.active ? (
+        <NotifiEmailInput
+          hideStartIcon={hideStartIcons?.email}
+          disabled={inputDisabled}
+          classNames={classNames?.NotifiEmailInput}
+          copy={{ label: inputLabels?.email }}
+        />
+      ) : null}
+      {inputSeparators?.emailSeparator?.content ? (
+        <div
+          className={clsx(
+            'NotifiInputSeparator__container',
+            inputSeparators?.emailSeparator?.classNames?.container,
+          )}
+        >
+          <div
+            className={clsx(
+              'NotifiInputSeparator__content',
+              inputSeparators.emailSeparator.classNames?.content,
+            )}
+          >
+            {inputSeparators?.emailSeparator?.content}
+          </div>
+        </div>
+      ) : null}
+      {data.contactInfo.sms.active ? (
+        <NotifiSmsInput
+          disabled={inputDisabled}
+          hideStartIcon={hideStartIcons?.sms}
+          classNames={classNames?.NotifiSmsInput}
+          allowedCountryCodes={allowedCountryCodes}
+          copy={{ label: inputLabels?.sms }}
+        />
+      ) : null}
+      {inputSeparators?.smsSeparator?.content ? (
+        <div
+          className={clsx(
+            'NotifiInputSeparator__container',
+            inputSeparators?.smsSeparator?.classNames?.container,
+          )}
+        >
+          <div
+            className={clsx(
+              'NotifiInputSeparator__content',
+              inputSeparators.smsSeparator.classNames?.content,
+            )}
+          >
+            {inputSeparators?.smsSeparator?.content}
+          </div>
+        </div>
+      ) : null}
+      {data.contactInfo.telegram.active ? (
+        <NotifiTelegramInput
+          disabled={inputDisabled}
+          hideStartIcon={hideStartIcons?.telegram}
+          classNames={classNames?.NotifiTelegramInput}
+          copy={{ label: inputLabels?.telegram }}
+        />
+      ) : null}
+      {inputSeparators?.telegramSeparator?.content ? (
+        <div
+          className={clsx(
+            'NotifiInputSeparator__container',
+            inputSeparators?.telegramSeparator?.classNames?.container,
+          )}
+        >
+          <div
+            className={clsx(
+              'NotifiInputSeparator__content',
+              inputSeparators.telegramSeparator.classNames?.content,
+            )}
+          >
+            {inputSeparators?.telegramSeparator?.content}
+          </div>
+        </div>
+      ) : null}
+      {data.contactInfo.discord?.active ? (
```

Comment:
have you added the `discord` and `twitter` fields to the contact info type? I can't seem to find it in this diff

Reply:
![image](https://user-images.githubusercontent.com/105258726/206332892-59d21b30-657b-45ea-9095-874fc82a04d6.png)
this? @juni-notifi